### DISCUSSION TITLE
Portainer Isn't available for rasberry pi 1?

### DISCUSSION THREAD

**Discussion by lutfor-diu** [MichaIng/DietPi#6702] [Q&A]
Portainer Isn't available for rasberry pi 1?

**Comment by MichaIng** [comment]
Yes, they provide containers for ARMv7 and above only: https://hub.docker.com/r/portainer/portainer-ce/tags?name=linux-arm

**Reply by lutfor-diu** [reply]
possible to add Yacht?

**Reply by MichaIng** [reply]
Like Portainer, Yacht is just a Docker container as well, so an installation is trivial: #6269

**Reply by nonnorm** [reply]
However, it looks like Yacht also has no support for ARMv6, so it can't be run on the RPi 1.

**Reply by MichaIng** [reply]
Ah you are right, as well ARMv7 and up only: https://hub.docker.com/r/selfhostedpro/yacht/tags

**Reply by MichaIng** [reply]
However, note that an RPi 1 or Zero is anyway too weak to run a lot of containers, for a GUI like Portainer or Yacht would be of real benefit. I would actually aim for installing software bare metal on these. The 100 MiB or more used by the container engine only, then the additional, not sure, 50 MiB or more for the GUI, and close to half of its whole RAM (if its even a 512 MB model, not 256 MB) is used already, leaving not much for the actual application(s) you want to use. Of course the swap file allows it, but performance will be underwhelming if the swap file is in such regular use.
